debian/changelog

```diff
@@ -1,3 +1,11 @@
+cloud-init (23.3.3-0ubuntu0~23.10.1) mantic; urgency=medium
+
+  * Upstream snapshot based on 23.3.3. (LP: #2040291).
```

Comment:
Unrelated, we also need to sync 23.3.x ChangeLog into main as we haven't pulled down are published changes yet into upstream/main

Reply:
yeah, I synced 23.3.2 when we did that one. I still need 23.3.3, but figured we should get the downstreams done first